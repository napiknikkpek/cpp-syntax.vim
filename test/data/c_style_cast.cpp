(int)a;
(const int)a;
(int&)a;
(int&&)a;
(int const*)a;
(int const&)a;
(int const* const) a;
(A) a;
(const A) a;
(A&)a;
(A&&)a;
(A const*)a;
(A const&)a;
(A const* const) a;
(A) foo(a);
(A) A::a;
(A)(a + b);
(A) + a;
A<decltype(foo((A)a))> a;
A<decltype(foo((int)a))> a;
