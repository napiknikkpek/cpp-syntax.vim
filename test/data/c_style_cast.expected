.zzz.x.
.aaaaa.zzz.x.
.zzz..x.
.zzz...x.
.zzz.aaaaa..x.
.zzz.aaaaa..x.
.zzz.aaaaa..aaaaa..x.
.t..x.
.aaaaa.t..x.
.t..x.
.t...x.
.t.aaaaa..x.
.t.aaaaa..x.
.t.aaaaa..aaaaa..x.
.t..fff.x..
.t..t..x.
.t..x...x..
.x....x.
t.aaaaaaaa.fff..t.x....x.
t.aaaaaaaa.fff..zzz.x....x.
